data/campaigns/Eastern_Invasion/scenarios/15_Return_to_Wesnoth.cfg

```diff
@@ -0,0 +1,977 @@
+        [message]
+            speaker=Dacyn
+            message= _ "And that is why my companions and I have spent the past year traveling far to the northeast, seeking (cough) a potent magical amulet that I now possess. If we can destroy the skeletal body, I wield the power to banish the lich’s long-tortured soul. This destruction will be permanent."
+        [/message]
+        [message]
+            speaker=Gweddry
+            message= _ "My King, I have seen this amulet used before. While I cannot attest to its reliability, it is certainly powerful enough to do what Dacyn claims."
+        [/message]
+        [message]
+            speaker=Halric
+            message= _ "Even assuming this magical voodoo works, how do you mean to destroy Mal-Ravanal’s body? The one chance we had was out of the lich’s own arrogance, and even that cost us four of our elite magi under better conditions. Now we’ve been cut off from Dan’Tonk and barely have enough men to hold the walls, much less sortie against the undead."
+        [/message]
+        [message]
+            speaker=Aeraeka
+            message= _ "Truly, have you three not seen the might of the armies encircling Weldyn? It is as if all of the dead warriors from all ages past have come back to attack us. There is no end to the columns of marching skeletons, the wailing ghosts, the infernal bats, the necromancers who raised them..."
+        [/message]
+        [message]
+            speaker=Owaec
+            message= _ "I pay no heed to the odds! I say we strike at Mal-Ravanal with all the might of Wesnoth. If we assault in force, we can punch through the enemy lines, isolate Mal-Ravanal in the chaos of battle, and blast the lich to oblivion!
+
+I will not let the devastation at the plains be the final legacy of the clans. We must attack and win back our glory!"
+        [/message]
+        [delay]
+            time=1000
+        [/delay]
+
+        [message]
+            speaker=Dacyn
+            message= _ "I have a (cough) superior suggestion. Mal-Ravanal’s armies are undoubtedly vast and unconquerable. Yet, I knew Ravan as a human. Back then, they were arrogant and exceedingly overconfident. I still believe this to be true, if not more so after gaining lichdom. (cough)(cough) If I were to challenge Mal-Ravanal to a duel, I believe they would choose to face me. That would be our opportunity."
+        [/message]
+        [message]
+            speaker=Owaec
+            message= _ "You call Mal-Ravanal arrogant, yet you seek to face such an overwhelming power in single combat? Even if you could claim victory with that foul amulet of yours, we cannot trust that lich to honor the terms of such a challenge. You would likely find yourself facing their entire army!"
+        [/message]
+        [message]
+            speaker=Dacyn
+            message= _ "We face their entire army regardless. (cough)(cough) The only difference is whether Mal-Ravanal is commanding in back or gloating in front."
```

Comment:
'commanding in the back' (or rear)

Reply:
I think I'd add the article to "front" as well, for consistency.

Reply:
Agreed.